File: src/main/java/requestreplyapi/Registry/Entry.java
package requestreplyapi.Registry;

import requestreplyapi.Commons.IAddress;

public class Entry implements IAddress
{
	private String destinationId;
	private int portNr;
	public Entry(String theDest, int thePort)
	{
		destinationId = theDest;
		portNr = thePort;
	}
	public String dest()
	{
		return destinationId;
	}
	public int port()
	{
		return portNr;
	}
}

File: ObjectRequestBrokerCS/ORB/requestreplyapi/entries/Entry.cs
namespace ORB.requestreplyapi.entries
{

	public class Entry : common.IAddress
	{
		private string destinationId;
		private int portNr;
		public Entry(string theDest, int thePort)
		{
			destinationId = theDest;
			portNr = thePort;
		}

		public Entry()
		{
		}

		public virtual string Dest()
		{
			return destinationId;
		}
		public virtual int Port()
		{
			return portNr;
		}

		public override string ToString()
		{
			return "Entry{" +
					"destinationId='" + destinationId + '\'' +
					", portNr=" + portNr +
					'}';
		}
	}
}